### PR TITLE
Citizen messaging UI fixes

### PR DESCRIPTION
**Pull request by mapehe**
#### Summary

Update the citizen messaging UI to better resemble the Figma design.

### CODE REVIEW

File: frontend/src/citizen-frontend/messages/ThreadView.tsx [hunk 1]
```diff
@@ -43,6 +45,10 @@ const MessageContent = styled.div`
   white-space: pre-line;
 `
 
+const ReplyToThreadContainer = styled.span`
+  padding-left: 28px;
+`
+
```
Comment (jaahon) [inline frontend/src/citizen-frontend/messages/ThreadView.tsx:51]: Super small thing, but you could remove this unnecessary `<span>` and apply this margin directly to the `InlineButton` component with `styled(InlineButton)`.

File: frontend/src/citizen-frontend/auth/accessibleFeatures.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      ) : (
+        <WrappedComponent {...props} />
+      )
+    ) : requireStrong ? (
+      <Redirect to={'/'} />
+    ) : (
+      refreshRedirect(getWeakLoginUri(props.location.pathname))
+    )
+  }
+}
```
Comment (jaahon) [inline frontend/src/citizen-frontend/auth/accessibleFeatures.tsx:35]: Was this file accidentally included in the PR?

**Review by jaahon — COMMENTED** [review]
Looks good! 👍

File: frontend/src/citizen-frontend/messages/ThreadView.tsx (hunk 1 above)
Comment (mapehe) [reply frontend/src/citizen-frontend/messages/ThreadView.tsx]: Nice!

File: frontend/src/citizen-frontend/auth/accessibleFeatures.tsx (hunk 2 above)
Comment (mapehe) [reply frontend/src/citizen-frontend/auth/accessibleFeatures.tsx]: Oops yeah lol

**Review by jaahon — APPROVED** [review]

**Pull request merged**